docs/source/guide/explanation/algorithms/object_detection/object_detection.rst

```diff
@@ -17,7 +17,49 @@ To be added soon
 Semi-supervised Learning
 ************************
 
-To be added soon
+Semi-SL (Semi supervised Learning) is a type of machine learning algorithm that uses both labeled and unlabeled data to improve the performance of the model. This is particularly useful when labeled data is limited, expensive or time-consuming to obtain.
```

Comment:
We have already the definition for the Semi-SL in the alforithms/index - we can omit it there